From f56b6fe825b776c30e445d2319488bec63551b8f Mon Sep 17 00:00:00 2001
From: Dominik Csapak <d.csapak@proxmox.com>
Date: Fri, 20 Jan 2017 10:16:09 +0100
Subject: [PATCH 08/10] add replaceable snippets in vnc.html

so that we can insert the username/csrftoken via search/replace

Signed-off-by: Dominik Csapak <d.csapak@proxmox.com>
---
 vnc.html | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/vnc.html b/vnc.html
index 8c3d9d9..a139e26 100644
--- a/vnc.html
+++ b/vnc.html
@@ -15,7 +15,7 @@
     or the fragment:
         http://example.com/#host=HOST&port=PORT&encrypt=1
     -->
-    <title>noVNC</title>
+    <title>[% nodename %] - Proxmox Console</title>
 
     <meta charset="utf-8" />
 
@@ -38,6 +38,12 @@
 
     <!-- this is included as a normal file in order to catch script-loading errors as well -->
     <script type="text/javascript" src="/novnc/app/error-handler.js"></script>
+    <script type="text/javascript">
+	if (typeof(PVE) === 'undefined') PVE = {};
+	PVE.UserName = '[% username %]';
+	PVE.CSRFPreventionToken = '[% token %]';
+	INCLUDE_URI='/novnc/include';
+    </script>
 
     <!-- begin scripts -->
     <!-- promise polyfills promises for IE11 -->
-- 
2.11.0

